resources/web/wwi/stream.js

```diff
@@ -2,17 +2,18 @@
 'use strict';
 
 class Stream { // eslint-disable-line no-unused-vars
-  constructor(url, view, onready) {
-    this.url = url;
+  constructor(wsServer, httpServer, view, onready) {
+    this.wsServer = wsServer;
+    this.httpServer = httpServer;
     this.view = view;
     this.onready = onready;
     this.socket = null;
     this.videoStream = null;
-    TextureLoader.setStreamingMode(true);
+    TextureLoader.setStreamingMode(true, this.httpServer);
```

Comment:
The `httpServer` in this class seems to be only used to setup the `TextureLoader`.
And this is the only call to `TextureLoader` in the `Stream` class.

I think it would be better to move this instruction in `webots.js` given that the textures are no longer streamed.
This way we can also remove the unneeded dependency of `Stream` on `TextureLoader`.
Also note that the first argument of `setStreamingMode` is no longer used and could be completely removed.